app/javascript/packs/timer.js

```diff
@@ -0,0 +1,209 @@
+  timers.append(timer);
+}
+
+function generateTimer(name, time) {
+  container = document.createElement('div');
+  container.id = `container${count}`;
+  container.classList.add('row');
+  container.classList.add('mt-3');
+
+  name_element = generateName(name);
+
+  time_element = document.createElement("p");
+  time_element.classList.add('my-auto');
+  time_element.classList.add('mr-2');
+  time_element.id = `timer${count}`
+  time_element.innerHTML = `${timeToString(time)}`;
+
+  button_start = generateStartButton();
+  button_pause = generatePauseButton();
+  button_delete = generateDeleteButton();
+
+  container.append(name_element);
+  container.append(time_element);
+  container.append(button_start);
+  container.append(button_pause);
+  container.append(button_pause);
+  container.append(button_delete);
+
+  count++;
+
+  return container;
+}
+
+function generateName(name_timer) {
+  let name = document.createElement('h5');
+  name.classList.add('my-auto');
+  name.classList.add('mr-5');
+  name.innerHTML = name_timer;
+  return name;
+}
+
+function generateStartButton() {
+  let btn = document.createElement("button");
+  btn.type = 'button';
+  btn.id = `btnStart${count}`;
+  btn.setAttribute('target', `timer${count}`);
+  btn.innerHTML = 'Start';
+  btn.classList.add('btn');
+  btn.classList.add('btn-success');
+  btn.addEventListener('click', function () { start(this); });
+  return btn;
+}
+
+function generatePauseButton() {
+  let btn = document.createElement("button");
+  btn.type = 'button';
+  btn.id = `btnStart${count}`;
+  btn.setAttribute('target', `timer${count}`);
+  btn.innerHTML = 'Pause';
+  btn.classList.add('btn');
+  btn.classList.add('btn-primary');
+  btn.addEventListener('click', function () { pause(this); });
+  return btn;
+}
+
+function generateDeleteButton() {
+  let btn = document.createElement("button");
+  btn.type = 'button';
+  btn.id = `btnStart${count}`;
+  btn.setAttribute('target', `timer${count}`);
+  btn.innerHTML = 'Delete';
+  btn.classList.add('btn');
+  btn.classList.add('btn-danger');
+  btn.addEventListener('click', function () { deleteTimer(this); });
+  return btn;
+}
+
+function showToastAlarm() {
+  setToast();
+  $('#liveToast').removeClass('hide');
+  $('#liveToast').addClass('fade show');
+}
+
+function setToast() {
+  let toastBody = document.getElementById('toastBody');
+  toastBody.innerHTML = "timer";
+}
+
+const getRemainingTime = deadline => {
+  let now = new Date(),
+    remainTime = (new Date(deadline) - now + 1000) / 1000,
+    remainSeconds = ('0' + Math.floor(remainTime % 60)).slice(-2),
+    remainMinutes = ('0' + Math.floor(remainTime / 60 % 60)).slice(-2),
+    remainHours = ('0' + Math.floor(remainTime / 3600 % 24)).slice(-2),
+    remainDays = Math.floor(remainTime / (3600 * 24));
+
+  return {
+    remainSeconds,
+    remainMinutes,
+    remainHours,
+    remainDays,
+    remainTime
+  }
+};
+
+const countdown = (deadline, elem, finalMessage, id) => {
+  console.log(id)
```

Comment:
Do you still need that console.log ? or it was just for testing?

Reply:
Thanks for the observation